controller/src/main/java/org/jboss/as/controller/descriptions/DefaultResourceDescriptionProvider.java

```diff
@@ -119,7 +120,17 @@ public ModelNode getModelDescription(Locale locale) {
         // Sort the attribute descriptions based on attribute group and then attribute name
         Set<String> attributeNames = registration.getAttributeNames(PathAddress.EMPTY_ADDRESS);
 
-        Map<AttributeDefinition.NameAndGroup, ModelNode> sortedDescriptions = new TreeMap<>();
+        //fix of WFCORE-1985 - attributes have to be sorted firstly by name
+        //case that two attributes of the same name are allowed is not possible, see error WFLYCTL0043
+        Comparator<AttributeDefinition.NameAndGroup> comparator = (x, y) -> {
```

Comment:
Could you please remove this lambda et replace it by an anonymous class please?

Reply:
@jmesnil it's done, I also renamed PR and issue to contain attributes as requested below